assets/sass/cds/_components.scss

```diff
@@ -253,6 +264,98 @@
       background: $primary-color;
     }
   }
+	list-style: none;
```

Comment:
Nit: formatting

Can this bit setting the values for .team-listing be moved up with the others on line 182?

Reply:
Actually, is this all duplicated from above?